apps/date-component/README.md

```diff
@@ -0,0 +1,84 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date). There is also a further explanation of useful date validations.
+
+## Getting started
+
+Add a step that displays a date of birth.
+
+```
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
+      next: '/confirm'
+    }
+  }
+```
+
+## Adding the Date component field
+
+Install the [hof-component-date](https://www.npmjs.com/package/hof-component-date) module
+
+In your field configuration file, create an instance of a data component. You can do this by calling the data component function with a field name and the options for the field. In this case, our field name is date-of-birth and in our options we set a required validator.
```

Comment:
You said it could either be referred to as a "date component" (casually - no hyphen) or "hof-component-date". What's your beef?